pkgs/by-name/gi/git-smash/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  installShellFiles,
+  makeWrapper,
+  fuzzyMatcherPackage ? fzf,
+  fzf,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "git-smash";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "anthraxx";
+    repo = "git-smash";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-NyNYEF5g0O9xNhq+CoDPhQXZ+ISiY4DsShpjk5nP0N8=";
+  };
+
+  cargoHash = "sha256-omITZMBWzYlHHim/IXNa1rtiwHqpgLJ5G9z15YvDRi0=";
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+
+  postFixup = ''
+    wrapProgram "$out/bin/git-smash" --prefix PATH : "${fuzzyMatcherPackage}/bin"
+  '';
+
+  postInstall = ''
```

Comment:
```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Reply:
Changed as proposed. Just to make sure I understand what's going on. This is to make sure we won't try to run the `git-smash` binary on an unsupported platform when cross compiling. Is that correct?